### PR TITLE
[Tests] add get_order_books tests

### PR DESCRIPTION
**Pull request by GuillaumeDSM**

### CODE REVIEW

**Review by Herklos — APPROVED** [review]
👍

**Comment by GuillaumeDSM** [comment]
up with limit tests

File: tests_additional/real_exchanges/real_exchange_tester.py [hunk 1]
```diff
@@ -47,6 +48,7 @@ class RealExchangeTester:
     REQUIRES_AUTH = False  # set True when even normally public apis require authentication
     MARKET_STATUS_TYPE = trading_enums.ExchangeTypes.SPOT.value
     HISTORICAL_CANDLES_TO_FETCH_COUNT = 650
+    DEFAULT_CUSTOM_BOOK_LIMIT = 50
```
Comment (Herklos) [inline tests_additional/real_exchanges/real_exchange_tester.py:51]: 👍

File: tests_additional/real_exchanges/real_exchange_tester.py [hunk 2]
```diff
@@ -229,6 +233,32 @@ async def inner_test_get_order_books(
                 raise AssertionError(
                     f"More empty book sides than expected: {max_empty_book_sides=} {empty_book_sides=}"
                 )
+            # with custom limit
+            books_by_symbol = await exchange_manager.exchange.get_order_books(symbols=None, limit=custom_limit)
+            self._ensure_book_custom_limit(
+                books_by_symbol, supports_custom_limit, expected_max_orders_by_side, custom_limit
+            )
+
+    def _ensure_book_custom_limit(
```
Comment (Herklos) [inline tests_additional/real_exchanges/real_exchange_tester.py:242]: 👍

**Review by Herklos — APPROVED** [review]

**Pull request merged**